src/github.com/getlantern/flashlight/android/main.go

```diff
@@ -0,0 +1,13 @@
+// main package holds references to the go_bindings package and is based on
```

Comment:
This file links our actual bindings with the Go's mobile package and is the basis for building the `.so` library. See https://godoc.org/golang.org/x/mobile/cmd/gobind